projects/cashmere/src/lib/accordion/accordion.component.spec.ts

```diff
@@ -0,0 +1,134 @@
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import {AccordionComponent} from './accordion.component';
+import {AccordionToolbarComponent} from './accordion-toolbar.component';
+import {AccordionModule} from './accordion.module';
+import {NoopAnimationsModule} from '@angular/platform-browser/animations';
+import {Component} from '@angular/core';
+import {By} from '@angular/platform-browser';
+
+@Component({
+    template: `
+        <hc-accordion
+            [triggerAlign]="alignVal"
+            [hideToolbar]="hideVal"
+            [toolbarTrigger]="triggerVal"
+            [(open)]="openVal"
+            (openStart)="onOpenStart($event)"
+            (opened)="onOpen($event)"
+            (closeStart)="onCloseStart($event)"
+            (closed)="onClosed($event)"
+        >
+            <hc-accordion-toolbar>Toolbar</hc-accordion-toolbar>
+            Content
+        </hc-accordion>
+    `
+})
+export class TestAccordionComponent {
+    alignVal: string = 'left';
+    hideVal: boolean = false;
+    triggerVal: boolean = true;
+    openVal: boolean = false;
+
+    onOpenStart: (event?: Event) => void = () => {};
+    onOpen: (event?: Event) => void = () => {};
+    onCloseStart: (event?: Event) => void = () => {};
+    onClosed: (event?: Event) => void = () => {};
+}
+
+fdescribe('AccordionComponent', () => {
```

Comment:
Remove f

Reply:
gah - thanks Joe.  I do that every time :-)